assets/js/modules/tagmanager/datastore/__factories__/builders.js

```diff
@@ -19,7 +19,8 @@
 /**
  * External dependencies
  */
-import { build, fake, sequence, oneOf } from '@jackfranklin/test-data-bot';
+import { build, sequence, oneOf, perBuild } from '@jackfranklin/test-data-bot';
+import faker from 'faker';
```

Comment:
We could probably reimplement `fake` to avoid rewriting all of the uses but there aren't that many so it's probably fine (and also more consistent with the documented use) 👍 

<details>
<summary><code>fake</code> polyfill</summary>

```js
function fake( func ) {
	return perBuild( () => func( faker ) );
}
```

</details>

Reply:
Yeah that's what I thought; there were so few it was nicer to just use the documented API.